loculus_values/environment_specific_values/staging.yaml

```diff
@@ -33,6 +33,12 @@ secrets:
       port: AgCVKaOX9s+R408WcJowEPaLZqz85/3nu6+q9DBExfFf/WArGvtPzoQIGX0n1ySgw+5skKMbcwhcc4jkbIrvKnphE0VexufFUyM3jFl9NuV+9dR2fSrEyUMgd3l5G5O0vyayL0ERgWi7DMcvpcbN7gfUrfzdWhNa7vhmAtWIZe+Hvff5lp2WKx8fVf4pPzDee0dfstCxQPdrF+N7/B6nN8k1kKiyDgxgx5YDSNpxSVWrpL2a3TfIlaubJTYplg62m0WSwPVcCNBnQaS7m75DwYLW/vTyAO3krNgXYqwaafPmkpZJmc6qT8RZE1e1sVizL7thfcJiZrOvZsxED9XYkEca6nQjyuV6EWubdymbxfhbmQy1AXsVXmCcbOrKGPv1GDui+EUw3dUiSD68+1VhUmy7EH/yYI4M2dZRtemJQPTiJhW8NfPqtZutkMdJGE5/dtjGTJ8OLeMiCz4TYyEqYwEFiUr8mfrLibdSdW6O9NAUP5niFx/5lNjNA19Xpu/8fJI0GUMY8s+UyPQC4W0xkJmof8zuZtq/n0r9PNSXIJulFQ6OTSaxMYqeHSG0xRVMoVs1/TPipjjyH8ssMtNK52ahWv4xNFEvoA6phLlXBAHBj4rHLVFDofnF9P2FZDQSmsevtAtc58wHvvI4Xe1skiKr4orUfZWVUaSuyQcHqub95XxwYQaMi7zGKfOZBcpiPvL8xFll
       username: AgAuZZ8Vs8bvPb1xqVEVNhQ5GpAL9f9gPfVNMCX1tT3NEXAPZn1/ujf6dl/ltaIeaO/DITWkC6YmODCcAvx5ABiwhYnDO7rY6V+UdIqTjQuPuoIf80L+kipf3ZTID/0nlS9yFMnSu++XdQqXJUGbPuOMyg9gMollWPyJJzTmQZQLYsXlvFrs0pifZzCbYwpRwyh3aCgMh6gZMdOlIGRR1n9xxY8oX2glx8+XdN3hoK3AEHf1E40/HZjLBGr3me+0gKvjw0kC3rQFJpHL917ODGB/3fJKT/rFF3CKMQLMkRV7ov6OUo+nKecDDEn3t8lDfWGPl3efKpZqyX5zIVz82r9q7qXWR3IluExZvp4gHncciuESO+LYONtS3faKppz3YBqEo14dyJtAVuph9evgfgl9yhtotfNUeu8DcXq9WrYbFhqnoSRrSOrslnAOgW+h0MJhbqoNTolaSqm/u/W45d9fKbPG53y9g8og/QnNF73sJLhq0P3NPMiZUUHhch5zcD3BLwr4RNd22ql0s0mxPtuRtXWWHgsQD5SZnCHSc5qfdcoDk2EkSyvNy1MzGhjt4itQUwawU98YvzU3jtiEnrkCkE3r0U5iyU+esajCam3HYorwSor9yjAIP1/P9ZSw7jKushqgh1FDXmZ0WXgvfXjbQKu5Ik63nEvJOGQ/gyuGc6QFR6Gh2YMZv85KEjvxbzydBGOmGly9zagaXKXtfmZzi+VCvgo=
+  ena-submission:
+    type: sealedsecret
+    clusterWide: "true"
+    encryptedData:
+      username: AgCpzdjJSQtr//NnBvQ4X/CGJTx0AX1fa0l1HxL1Y/i1dT74coFR0yAvEEvbOCaTXdKkn6V+1S+jWn64FwCUipjavz1gYvbeGtE9L3LwQ4eA0qt5JejEyPJn8fa+sgS41MLymP3UHn0R8A71wrrX3hrSrxLFa0WAmGhQ8RCayKiQhxHQpluzPzFsmFWodK9lq2jNnn/2V0x41PHtWh/FDf+HXVNNBJTyK5Zp6Wns5QQEdMrsk4YcVC0LGjM6ayreMN++JwY66MsokEC1llVvp+H8WnIzUOGBfeAon70FlXecX07rgwO9eA+v4icpXiR2u6hsAxiRM9LrM5QtwMjI4l37+V4NHiCTPk1Odjoet/HxYmeP3dDewvFIVEtgxUCEo/DYmD8uWSkKIjc9rNOB34oIEdT95nJlYhKp5HUdu7zfGFXiESsVcyz2sePxPgP1yuBNfRVScjd3FagXm/3u9AsfzTL50eBY2oQNl+ymqJZ1kvXZhQXf2WWSUrZgs2TKqP5WgeOdAqr6no/DkQyv/RD4dDCHWFKoOndiW0JXxoO8i2n34Zoy4GzxorwbUQteY89Is6xgRwglyV2Wg8HUUG+MPOml+0dDcoNOn7GsFqCKAjmNQ0JLU0Twbjb6KZlmLHWHwpECWV0+Ea6YCf7GPO/koQl3Uh/FEzvkeEaDLSkBfSVuDfGK9oepuFGHC8kmFxR56Oc/TkkUsOZz3g==
```

Comment:
Do we actually want this on staging?

Reply:
Ah I guess at least for now while we figure stuff out

Reply:
Thanks sorry I was gonna reply but forgot!

Cornelius and I discussed first submitting to the ENA dev instance on staging using this account (and hopefully the check that I added to the code will prevent staging ever talking to ENA prod), its probably not necessary as we have been doing this on loculus previews but we can wipe the db on staging easily now so it sounds like an "easy" check.

But yes - probably best to just delete the credentials afterwards from staging to avoid any unexpected issues.